react/features/base/conference/reducer.js

```diff
@@ -209,7 +209,11 @@ function _lockStateChanged(state, action) {
         return state;
     }
 
-    return setStateProperty(state, 'locked', action.locked || undefined);
+    return setStateProperties(state, {
+        locked: action.locked || undefined,
+        lockedLocally: Boolean(state.lockedLocally && action.locked),
```

Comment:
I'm bothered by the lack of symmetry in having `lockedLocally` and not having `lockedRemotely`. I think I'd have been more at peace with 1 single `locked` state which evaluates to an enumerable set of values such as:
- If `locked` is falsy, then the conference is unlocked.
- If `locked` is truthy, then the conference is locked. And if I as the API consumer care about how it's locked, I can go beyond truthy by examining more specifics of the value such as is it equal to LOCKED_LOCALLY or LOCKED_REMOTELY.
WDYT, @virtuacoplenny?